bundles/ch.elexis.core/src/ch/elexis/core/services/IOrderService.java

```diff
@@ -30,4 +41,85 @@ public interface IOrderService {
 	 * @since 3.6 return null on invalid order amount
 	 */
 	public @Nullable IOrderEntry addRefillForStockEntryToOrder(IStockEntry ise, IOrder order);
+
+	/**
+	 * Calculate the number of consumed articles (billables) on a given date.
+	 * 
+	 * @param date      the selected date
+	 * @param mandators optional list of mandators to limit the query
+	 * @return a map of article to total consumed amount
+	 */
+	public Map<IArticle, Integer> calculateDailyConsumption(LocalDate date, List<IMandator> mandators);
+
+	/**
+	 * Reduce open order entries for the given article if it was over-ordered.
+	 * 
+	 * @param order    the current order
+	 * @param article  the article to reduce
+	 * @param reduceBy amount to reduce
+	 */
+	public void reduceOpenEntries(List<IOrder> order, IArticle article, int reduceBy);
+
+	/**
+	 * Adds the specified article quantities to existing open entries within the
+	 * provided orders, or creates new entries in the given order if no matching
+	 * open entry is found.
+	 * <p>
+	 * For each article in {@code entriesToAdd}, the method attempts to find an open
+	 * {@link IOrderEntry} with the same article in {@code existingOrders}. If
+	 * found, the quantity is increased. If not, a new entry is created in
+	 * {@code createOrder}.
+	 * <p>
+	 * If a {@code mandator} is provided, it is used to determine the preferred
+	 * stock entry when creating a new order entry.
+	 *
+	 * @param existingOrders the list of existing orders to search for matching open
+	 *                       entries
+	 * @param createOrder    the order in which new entries are created when no
+	 *                       matching open entry exists
+	 * @param entriesToAdd   a map of articles and their quantities to be added
+	 * @param mandator       the active mandator, used to resolve preferred stock
+	 *                       entries (may be {@code null})
+	 */
+	public void addOrCreateOrderEntries(List<IOrder> existingOrders, IOrder fallbackOrder,
```

Comment:
Hier steht noch `fallbackOrder`, in der doc heist der `param` aber `createOrder`. Bitte auf `createOrder` umbenennen.